consensus/polybft/event_tracker_test.go

```diff
@@ -13,17 +14,28 @@ import (
 )
 
 type mockEventSubscriber struct {
+	lock sync.RWMutex
 	logs []*ethgo.Log
 }
 
 func (m *mockEventSubscriber) AddLog(log *ethgo.Log) {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
 	if len(m.logs) == 0 {
 		m.logs = []*ethgo.Log{}
 	}
 
 	m.logs = append(m.logs, log)
 }
 
+func (m *mockEventSubscriber) getLogs() []*ethgo.Log {
+	m.lock.RLock()
+	defer m.lock.RUnlock()
+
+	return m.logs
```

Comment:
It won't work with slices, if we can overwrite the slice elements for example? Can we have like thread safe Len() method instead?

Reply:
Updated with https://github.com/0xPolygon/polygon-edge/pull/965/commits/1ce9901abef40e7b193850fe78678855d677bef1